### PR TITLE
Add 2 VM APRL recommendations for maintenance sensitive workload

### PR DESCRIPTION
**Pull request by pinyuko**
# Overview/Summary
Add 2 VM APRL recommendations for maintenance sensitive workload
VM-27 - Use Azure Boost VMs for Maintenance sensitive workload 
VM-28 - Enable Scheduled Events for Maintenance sensitive workload VMs

Edit one typo for VM-26. It should be "Application Resilience" instead of "Application Resiliency" based on https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing/#recommendation-categories

## Related Issues/Work Items

Replace this with a list of related GitHub Issues and/or ADO Work Items (Internal Only)

- To associate a GitHub Issue, use a [key word](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) preceded with the GitHub issue number.
- To associate an ADO Work Item, use the key word `AB#` succeeded with the [ADO Work Item ID](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## This PR fixes/adds/changes/removes

1. Add VM-27 - Use Azure Boost VMs for Maintenance sensitive workload 
2. Add VM-28 - Enable Scheduled Events for Maintenance sensitive workload VMs
3. Edit one typo for VM-26. It should be "Application Resilience" instead of "Application Resiliency" based on https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing/#recommendation-categories
4. Create blank (// under-development) query file for VM-27 and VM-28

### Breaking Changes

1. N/A

## As part of this Pull Request I have

- [x] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [x] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [x] Ensured PR tests are passing
- [ ] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Review by oZakari — CHANGES_REQUESTED** [review]
Hi @pinyuko, could you please fix the lint errors related to formatting? Otherwise, looks good to me.

**Review by oZakari — APPROVED** [review]

File: docs/content/services/compute/virtual-machines/_index.md [hunk 1]
```diff
@@ -728,3 +731,54 @@ All VMs that are members or a SQL Always-on cluster must use the same VM Sku, sa
...
+
+**Impact: Medium**
+
+**Guidance**
+
+If the workload is Maintenance sensitive, please consider using Azure Boost compatible VMs. Azure Boost is designed to lessen the impact on customers when Azure maintenance activities occur.
+
+**Resources**
+
+- [Microsoft Azure Boost](https://learn.microsoft.com/en-us/azure/azure-boost/overview)
```
Comment (ejhenry) [inline docs/content/services/compute/virtual-machines/_index.md:747]:
```suggestion
- [Microsoft Azure Boost](https://learn.microsoft.com/azure/azure-boost/overview)
```

File: docs/content/services/compute/virtual-machines/_index.md [hunk 2]
```diff
@@ -728,3 +731,54 @@ All VMs that are members or a SQL Always-on cluster must use the same VM Sku, sa
...
+
+**Impact: Medium**
+
+**Guidance**
+
+If the workload is Maintenance sensitive, please enable Scheduled Events. Scheduled Events is an Azure Metadata Service that gives your application time to prepare for virtual machine maintenance. It provides information about upcoming maintenance events (for example, reboot) so that your application can prepare for them and limit disruption. It's available for all Azure Virtual Machines types, including PaaS and IaaS on both Windows and Linux.
+
+**Resources**
+
+- [Azure Metadata Service: Scheduled Events for Linux VMs](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/scheduled-events)
```
Comment (ejhenry) [inline docs/content/services/compute/virtual-machines/_index.md:772]:
```suggestion
- [Azure Metadata Service: Scheduled Events for Linux VMs](https://learn.microsoft.com/azure/virtual-machines/linux/scheduled-events)
```

File: docs/content/services/compute/virtual-machines/_index.md [hunk 3]
```diff
@@ -728,3 +731,54 @@ All VMs that are members or a SQL Always-on cluster must use the same VM Sku, sa
...
+- [Azure Metadata Service: Scheduled Events for Windows VMs](https://learn.microsoft.com/en-us/azure/virtual-machines/windows/scheduled-events)
```
Comment (ejhenry) [inline docs/content/services/compute/virtual-machines/_index.md:773]:
```suggestion
- [Azure Metadata Service: Scheduled Events for Windows VMs](https://learn.microsoft.com/azure/virtual-machines/windows/scheduled-events)
```

File: docs/content/services/compute/virtual-machines/_index.md [hunk 4]
```diff
@@ -728,3 +731,54 @@ All VMs that are members or a SQL Always-on cluster must use the same VM Sku, sa
...
+- [Monitor scheduled events for your Azure VMs](https://learn.microsoft.com/en-us/azure/virtual-machines/windows/scheduled-event-service)
```
Comment (ejhenry) [inline docs/content/services/compute/virtual-machines/_index.md:774]:
```suggestion
- [Monitor scheduled events for your Azure VMs](https://learn.microsoft.com/azure/virtual-machines/windows/scheduled-event-service)
```

**Review by ejhenry — APPROVED** [review]
LGTM

**Pull request merged**